src/query/sql/src/planner/binder/ddl/index.rs

```diff
@@ -220,12 +226,18 @@ impl Binder {
         Ok(plan)
     }
 
-    fn check_index_support(query: &Query) -> Result<()> {
+    fn check_index_support(query: &Query, has_now_func: bool) -> Result<()> {
+        let now_func = "now";
```

Comment:
Better way: get the function signature and check if `is_deterministic`. Because there are other functions are not deterministic such as `random`.

Reply:
Oh yep, I forgot to modify this, nice capture.